### ISSUE TITLE
Num epochs and batch size

### ISSUE THREAD

**Issue by LedaguenelArthur** [kylemin/HEX-graph#2]
Hi @kylemin,

Another question on the technical details of your experiments 😛 
Do you remember the hyper-parameters of training, like the batch size and the number of epochs ?

Best regards,
Arthur Ledaguenel

**Comment by kylemin** [comment]
Hi again,

Honestly, I do not remember all the details, but for the experiments using the VGG-fc7 output, I think I used a batch size of 2048. For CIFAR-100, I used a smaller number like 128 or 256. For all the experiments, I optimized the results using the validation loss.

I hope this helps!
Kyle

**Comment by LedaguenelArthur** [comment]
Thanks a lot Kyle for all these answers !

I'm trying to understand if my compute time is "normal" or if there are still things that can be heavily optimized in my code, that's I was wondering what batch size you were using.

Do you hve any idea for how long or how many epochs did you train the network before reaching the final performances ?


Again I realize that it has been a while since you worked on this project so thx a lot for the help,
Best regards,
Arthur

**Comment by kylemin** [comment]
You're welcome!
As far as I remember, the experiments using the VGG-fc7 output took less than 10 epochs. But I am not sure about CIFAR-100.

Thank you,
Kyle
